### PR TITLE
Add Bulk Check

### PR DESCRIPTION
**Pull request by gemanor**

### CODE REVIEW

**Comment by gemanor** [comment]
image

**Comment by gemanor** [comment]
These tests are not related to the change and fail in Main too
image

**Comment by obsd** [comment]
update the version to  `version="2.1.2"` in setup.py file

File: permit/enforcement/enforcer.py [hunk 1]
```diff
@@ -42,6 +48,125 @@ def context_store(self):
         """
         return self._context_store
 
+    async def bulk_check(
+        self,
+        checks: list[CheckQuery],
+        context: Context = {},
```
Comment (obsd) [inline permit/enforcement/enforcer.py:54]: Why the context isn't part of the checks object?

**Review by obsd — APPROVED** [review]

**Pull request merged**